src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -163,11 +161,15 @@ export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProp
         this.setState({ showNeedsSettingsContent: true });
     }
 
-    private openDropdown = (event): void => {
-        this.setState({ target: event.currentTarget, isContextMenuVisible: true });
-    };
+    private showToastWithSuccessMessage(): void {
+        this.setState({ showingCopyToast: true, toastText: 'Failure details copied.' });
+    }
+
+    private hideToast(): void {
+        this.setState({ showingCopyToast: false, toastText: '' });
+    }
 
-    private dismissDropdown(): void {
-        this.setState({ target: null, isContextMenuVisible: false });
+    private showToastWithFailureMessage(): void {
+        this.setState({ showingCopyToast: true, toastText: 'Failed to copy failure details. Please try again.' });
```

Comment:
Can we check with Fer and Chelsea since this is a UI change

Reply:
I am also curious if we have asked Fer and Chelsea what they think of this failure message.

Reply:
yeah, I've checked with Fer yesterday. This is the text she told me yesterday.